app/messaging/application/submit-claim.js

```diff
@@ -67,7 +66,7 @@ const submitClaim = async (message) => {
     }
   } catch (err) {
     appInsights.defaultClient.trackException({ exception: err })
-    console.error(`failed to submit claim for request ${JSON.stringify(message.body)}`, err)
+    console.error('failed to submit claim for request', err)
```

Comment:
should we be using logger here?

Reply:
We should but this one of those I mentioned in my rant in our Slack channel. The method itself is abstracted away behind a couple of layers so getting a request logger down here is a bit difficult. Didn't want to turn this into a massive PR so will return to try and flatten this a bit so we can either wrap in a try catch or pass the request/logger down only one level.